src/internal/styles/forms/mixins.scss

```diff
@@ -238,8 +238,8 @@
     inset-inline-start: 0px;
     inset-inline-end: 0px;
     background-color: transparent;
-    border-block: awsui.$border-field-width solid awsui.$color-border-dropdown-container;
-    border-inline: awsui.$border-field-width solid awsui.$color-border-dropdown-container;
+    border-block: awsui.$border-width-field solid awsui.$color-border-dropdown-container;
```

Comment:
I would expect dropdown container and popover to have their own token, not use form field.